cdisc_rules_engine/config/config.py

```diff
@@ -20,10 +20,6 @@ def __new__(cls):
                 "REDIS_HOST_NAME",
                 "REDIS_ACCESS_KEY",
                 "CDISC_LIBRARY_API_KEY",
-                "CDISC_LIBRARY_URL",
-                "DICTIONARIES_TABLE_NAME",
-                "WHODRUG_TERMS_TABLE_NAME",
-                "MEDDRA_TERMS_TABLE_NAME",
                 "DATA_SERVICE_TYPE",
```

Comment:
CDISC_LIBRARY_URL got changed to a hardcoded value, table names should not be set here

Reply:
Does it makes sense to keep CDISC Library API customizable? In Parexel we work with the CDISC Library via a mirror.

Reply:
It will be a separate PR, so it is more like a question whether we should do it later or not.

Reply:
Yes, we can implement it later in a separate PR